src/config.ts

```diff
@@ -0,0 +1,38 @@
+import type { MergeableRecord} from './util/objects'
+import {deepMergeObject} from './util/objects'
+import path from 'path'
+import fs from 'fs'
+import {log} from './util/log'
+
+export interface FlowrConfigOptions extends MergeableRecord {
+	ignoreSourceCalls: boolean
+}
+
+export const defaultConfigFile = 'flowr.json'
+export const defaultConfigOptions: FlowrConfigOptions = {
+	ignoreSourceCalls: false
+}
+export const config = parseConfigOptions(process.cwd())
```

Comment:
the over-engineer part of me wants to allow cli options for this but i think it is fine for now :)

Reply:
But at least the config file name should be configurable :eyes: